eventbus/src/main/java/pl/edu/uj/kimage/eventbus/VertXEventBus.java

```diff
@@ -0,0 +1,36 @@
+package pl.edu.uj.kimage.eventbus;
+
+public class VertXEventBus implements EventBus {
```

Comment:
First cause Vertx isn't only one library which provides event bus. Second, for now we foresee the possibility of adding another event bus so the implementation name should reflect responsibilty or category. For example if we will have to add a second implementation of VertX event bus but with configuration for communication between plugins, what will be its name? VertX2EventBus?